tests/jballerina-unit-test/src/test/resources/test-src/expressions/ternary/ternary-expr-negative.bal

```diff
@@ -11,4 +11,15 @@ public function main(string... args){
 
 function test1 (int value) {
     string s1 = value > 40 ? true : false ? "morethan40" : "lessthan20";
-}
\ No newline at end of file
+}
+
+function test2() {
+    int[] a = [];
+    boolean True = true;
```

Comment:
Variables names should start in lower case. Let's use `trueVal` or arbitrary names like `x`. 

Please check wherever applicable.